### ISSUE TITLE
Images per row 

### ISSUE THREAD

**Issue by sT0pe** [benhowell/react-grid-gallery#122]
<Gallery
      images={images}
      enableImageSelection={false}
      backdropClosesModal={true}
      rowHeight={100}
      maxRows={2}
      margin={0}
/>

 thumbnail 100x100
 rowHeight={100}

### Expected behaviour
gallery wrapper width < 300 than third image goes to second row

### Actual behaviour
Third image goes down only after width <=200

So is it possible do not compress images, but if there is no enough space just move to the next row?

**Comment by alandroid** [comment]
I am having trouble with 'images per row', im trying to do `<Gallery maxRows={2} ... /> `to no effect :(

**Comment by mattg66** [comment]
I am also looking for this, I am not hopeful though as the project has been abandoned

**Comment by itoldya** [comment]
Hello @sT0pe and @mattg66 👋 

Do you want thumbnails to keep the original aspect ratio, i.e. no edges have been cut? React-grid-gallery uses compression for the gallery to have all rows completely filled. It's the main idea of the library. If it hasn't been done gallery may look like this on some resolutions. Is that behavior your ask for?

<img width="538" alt="Снимок экрана 2022-08-10 в 20 52 25" src="https://user-images.githubusercontent.com/977637/183982419-b76a2de6-d833-4cb5-9508-c9e8a1b0d2b8.png">



The original gallery used on photos.google.com doesn’t do compression of images. Instead, it can vary row height to avoid incomplete rows. I think it's interesting and we may implement this strategy as an option in the next releases.

Can you please clarify your use case?

**Comment by sT0pe** [comment]
Hi @itoldya, yes that is exactly what I was looking for in 2019. Would be nice to have an option for such behavior.
